xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -262,11 +262,33 @@ bool CGUIWindowPVRRecordings::OnMessage(CGUIMessage &message)
           {
             case ACTION_SELECT_ITEM:
             case ACTION_MOUSE_LEFT_CLICK:
+              switch(CSettings::GetInstance().GetInt(CSettings::SETTING_MYVIDEOS_SELECTACTION))
+              {
+                case SELECT_ACTION_CHOOSE:
+                  OnPopupMenu(iItem);
+                  bReturn = true;
+                  break;
+                case SELECT_ACTION_PLAY_OR_RESUME:
+                  PlayFile(m_vecItems->Get(iItem).get(), false, true);
```

Comment:
Good point for <code>case ELECT_ACTION_RESUME</code> below, where it is used **multiple times in one case**. Will change this.